src/mlagility/analysis/util.py

```diff
@@ -1,16 +1,12 @@
-import os
```

Comment:
Adding clean as a cli arg sounds very dangerous if we are not checking whether or not the path given to us actually corresponds to an mlagility cache folder. 

Just imagine the following command if the user is NOT inside an mlagility cache folder:
`benchit clean -d . `

How do you feel about adding something like a hidden file `.mlagility` on all cache files, so we know that the path corresponds to something we can actually safely delete/clean?

Reply:
Done! .mlacache in cache dirs and .mlabuild in build dirs